doc/source/whatsnew/v0.25.0.rst

```diff
@@ -401,6 +401,8 @@ If installed, we now require:
 +-----------------+-----------------+----------+
 | pytz            | 2015.4          |    X     |
 +-----------------+-----------------+----------+
+| python-dateutil | 2.6.1           +   X
```

Comment:
can you space over one (i know this is pedantic, but otherwise won't be see right away)